### DISCUSSION TITLE
I am excited to be contributing to this project

### DISCUSSION THREAD

**Discussion by ChisomNwankwo** [ModECI/MDF#506] [General]
Hello Everyone, My name is Nwankwo Chisom. I am from Nigeria and I am excited to be contributing to this project. I am open to learn and improve in the process as well...

**Comment by OlawumiSalaam** [comment]
Welcome @ChisomNwankwo. Happy to collaborate with you

**Comment by pgleeson** [comment]
Hi @ChisomNwankwo! See https://github.com/ModECI/MDF/discussions/509 for more details.

**Comment by dawit-andargachew** [comment]
Hey Everyone, I'm Dawit. Glad to be here.

Thanks to this project, I've found an exciting opportunity to work on AI/ML-related tasks. With my prior experience in AI/ML internships and UI development, I'm eager to contribute my skills to this project.

I am getting my hands dirty on Jupyter Notebook by experimenting with MDF models. I have been tinkering with Streamlit as well to create some user interfaces. It's been a fun experience, especially considering my previous experience with Streamlit.

**Comment by akriti-dhurandher** [comment]
Hello everyone, I am Akriti Dhurandher and I am excited to be contributing to this project

**Comment by codinghappiness-web** [comment]
Hi @dawit-andargachew , I will be happy to collaborate with you
